gallery/psget/repository/bootstrapping_nuget_proivder_and_exe.md

```diff
@@ -6,36 +6,41 @@ author:  jpjofre
 ms.prod:  powershell
 keywords:  powershell,cmdlet,gallery
 ms.date:  2016-10-14
-contributor:  manikb
+contributor:  manikb, mgreenegit
 title:  bootstrapping_nuget_proivder_and_exe
```

Comment:
DEFINITIVE ANSWER:
Change the title. 
![image](https://cloud.githubusercontent.com/assets/6953993/26019949/5f670cfc-372e-11e7-9b8e-2a3331248bfc.png)

>Field: Title. Description: Title of the article. Purpose: (SEO) This is the title that gets displayed on a search engine results page. Only the first 65 characters will be displayed, so keep it brief. Use your main SEO keyword for this topic here.